pkg/reconciler/ingress/resources/virtual_service.go

```diff
@@ -211,6 +211,7 @@ func makeVirtualServiceRoute(hosts sets.String, http *v1alpha1.HTTPIngressPath,
 		Headers: h,
 	}
 
+	route.Retries = &istiov1alpha3.HTTPRetry{}
```

Comment:
This way you recreate the HTTPRetry object (c.f my comment earlier). But it's fine — this is not data path.